source/dub/platform.d

```diff
@@ -42,6 +42,7 @@ enum string platformCheck = q{
 	version(Android) ret ~= "android";
 	version(Cygwin) ret ~= "cygwin";
 	version(MinGW) ret ~= "mingw";
+	version(WebAssembly) ret ~= "wasm";
```

Comment:
This is good for completeness sake, but it won't do much, since it only ever runs when compiling dub itself to WebAssembly.

Reply:
Nope, this is also mixed into a platform probe file. The compiler compiles that one, using dub's `--arch` as LDC `-mtriple`, so that the BuildPlatform will feature a `wasm` platform.